src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -169,15 +133,41 @@ class SeekBar extends Slider {
    *          The current percent at a number from 0-1
    */
   update(event) {
-    // if the offsetParent is null, then this element is hidden, in which case
-    // we don't need to update it.
-    if (this.el().offsetParent === null) {
```

Comment:
This never did anything since seek bar is always visible as far as the dom is concerned due to screen readers.